packages/fxa-settings/config/webpack.config.js

```diff
@@ -355,6 +356,9 @@ module.exports = function (webpackEnv) {
       fallback: {
         fs: false,
         path: false,
+        buffer: require.resolve('buffer/'),
+        crypto: require.resolve('crypto-browserify'),
+        stream: require.resolve('stream-browserify'),
```

Comment:
Is this related to crypto-relier?

Reply:
I presume its because fxa-crypto-relier was pulling these in, and now webpack needs a direct reference itself?